src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -365,30 +365,6 @@ where
 
                 while let Some((cap, data)) = input.next() {
                     for (message, _, _) in data.iter() {
-                        let status = match &message {
-                            Ok(_) => HealthStatusUpdate::running(),
```

Comment:
This is where we would previously transition an export to "running" status once the first data update flowed through. The source operators now emit the "running" statuses for all exports, so afaict this code is not needed anymore.

Btw, I initially tried a smaller fix in this operator, where I would emit a "running" status in response to frontier updates instead of data updates. Unfortunately this doesn't work because we are still in the "source time" part of the dataflow, so when the upstream has no activity, there won't be frontier updates either.